reference-architecture/vmware-ansible/roles/openshift-auth-proxy-setup/tasks/main.yaml

```diff
@@ -0,0 +1,39 @@
+- name: Generate Certificate
+  command: "openshift start --public-master={{openshift_master_cluster_public_hostname}}:{{console_port}} --write-config=/etc/origin"
```

Comment:
Is this being done to get around the chicken/egg of having the certs prior to configuring the auth provider?